presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixCaching.java

```diff
@@ -328,7 +328,7 @@ public void testGetBlockLocations()
         cachingFileSystem = getCachingFileSystem();
 
         FileStatus file1 = new FileStatus(3, false, 0, 3, 0, new Path("aaa"));
-        FileStatus file2 = new FileStatus(3, false, 0, 3, 0, new Path("bbb"));
+        FileStatus file2 = new FileStatus(3, false, 0, 3, 0, new Path("zzzz"));
```

Comment:
just curious, how this relates to version update?

Reply:
The hashing function changed with https://github.com/qubole/rubix/commit/7e797a544d868440b753326cf03712881ed00ba2
With the new function, the paths in this test started mapping to the same worker node.
So I changed the path to get the test to pass :)

Reply:
How many "apparent nodes" do we have here?
can the new hashing function increase collusions % ?

Reply:
This test was meant to test that block locations are actually correctly propagated.

Reply:
The new hashing function is expected to be better as it switches from guava's MD5+consistent hash to Metro hash + consistent hash from a library which has a better implementation of consistent hashing https://github.com/ishugaliy/allgood-consistent-hash.
Since there are only 2 nodes here, I don't think we can conclude much about quality of hash function here.